src/modal/modal-backdrop.ts

```diff
@@ -1,5 +1,5 @@
 import {Component} from '@angular/core';
 
-@Component({selector: 'ngb-modal-backdrop', template: '', host: {'class': 'modal-backdrop fade show'}})
+@Component({selector: 'ngb-modal-backdrop', template: '', host: {'class': 'modal-backdrop show'}})
```

Comment:
Not sure about removing the `fade`class and leaving `show`. Take a look at BS styles:

![screen shot 2018-03-16 at 16 02 41](https://user-images.githubusercontent.com/8074436/37528010-c1678b78-2933-11e8-9aee-4048df2a5c4c.png)

Should we remove both?

Reply:
Yeh, probably... I was just afraid that people might be using `show` class for some reason (DOM manipulation). But in this case I should not be removing fade...

I will revert all this for now and only do inline style => `d-block` in this PR